### ISSUE TITLE
Problems with node 0.10.24

### ISSUE THREAD

**Issue by sweebee** [saberone/pimatic-smartmeter#14]
Some people have problems with serialport 2.0.6 on the older node version. While for others it works fine.

Could you create a separate branch for pimatic 0.9.x ? and publish both to npm. One for pimatic 0.8 and one for 0.9.

0.8 needs serialport 1.4.5:

```
"dependencies": {
    "serialport": "~1.4.5"
  },
```

**Comment by saberone** [comment]
Ok will do the branching. But what about the separate npm package name? Maybe it's better to "upgrade" the default branch, and create a compatibility branch with pimatic-classic ;) as npm package name?

**Comment by sweebee** [comment]
Yes, there must be a separate branch for pimatic 0.8 in npm. So people on 0.8 will automatically get the "older" one and 0.9 users get the newer one. Like pimatic-homeduino, pimatic-ping does.

**Comment by saberone** [comment]
Hmm.. I made the branch and publish that version to npm as pimatic-smartmeter-old.

**Comment by sweebee** [comment]
Its possible to keep the same name. It knows which pimatic version it has and it will install automatically the right one. Don't ask me how to do it :P but sweetpi does the same with homeduino and ping. Maybe you can ask him.

**Comment by mwittig** [comment]
Suggested procedure:
- Use last commit for version 0.0.16 to create branch for the plugin to be installed with pimatic 0.9, e.g. name the branch v0.9.x
- Inside the branch bump the version number, e.g. to 0.9.0
- Now you can publish this version for pimatic beta as follows: `npm publish --tag beta`

Back on the master branch you can prepare a version with the fixed serialport version using the original package name, e.g. 0.0.17  and publish it as a normal version.
